website/public/index.ejs.html

```diff
@@ -616,24 +619,32 @@ <h2 id="linter-feature-comparison">Features</h2>
                   title="with plugin"
                   class="footnote-marker"
                   >&nbsp;<a href="https://typescript-eslint.io/">†</a></span
-                >
+                ><span class="show-mobile"><br />typescript</span>
               </td>
               <td class="bad"><span aria-label="no">❌</span></td>
-              <td class="good"><span aria-label="yes">✅</span></td>
+              <td class="good">
+                <span aria-label="yes">✅</span
+                ><span class="show-mobile"><br />typescript</span>
+              </td>
               <td class="good"><span aria-label="yes">✅</span></td>
             </tr>
 
             <tr>
               <th>lint rules</th>
-              <td class="meh quick-lint-js"><a href="errors/">minimal</a></td>
+              <td class="meh quick-lint-js">
+                <a href="errors/">minimal</a
+                ><span class="show-mobile"><br />lint rules</span>
+              </td>
               <td class="good">
-                <a href="https://eslint.org/docs/rules/">zillions</a>
+                <a href="https://eslint.org/docs/rules/">zillions</a
+                ><span class="show-mobile"><br />of lint rules</span>
```

Comment:
This "of lint rules" label seems to show even on the desktop view. Shouldn't this be visible only in the mobile view?